test/code/plugins/out_oms_changetracking_file_systest.rb

```diff
@@ -0,0 +1,179 @@
+require 'fluent/test'
+require_relative ENV['BASE_DIR'] + '/source/ext/fluentd/test/helper'
+require_relative ENV['BASE_DIR'] + '/source/code/plugins/out_oms_changetracking_file'
+require_relative 'omstestlib'
+require_relative 'out_oms_systestbase'
+
+class TestableOutChangeTrackingFile < Fluent::OutChangeTrackingFile
```

Comment:
Just a thought - Is there a way to add a test (a more functional - E2E test) which uploads the blob content as well?